AdvancedDLSupport/NativeLibraryBuilder.cs

```diff
@@ -117,6 +120,28 @@ public NativeLibraryBuilder
             PathResolver = pathResolver ?? new DynamicLinkLibraryPathResolver();
         }
 
+        /// <summary>
+        /// Overrides the default symbol loader for this instance of <see cref="NativeLibraryBuilder"/>.
+        /// </summary>
+        /// <param name="factory">Factory to create the overriding symbol loader.</param>
+        /// <returns>This instance of <see cref="NativeLibraryBuilder"/>.</returns>
```

Comment:
Write "This instance, with the symbol loader overridden." instead. We know it's a NativeLibraryBuilder.